great-idea/css/index-stretch.css

```diff
@@ -0,0 +1,468 @@
+		justify-content: center;
+		margin: 0 auto;
+	}
+
+	.services-header-img {
+		width: 100%;
+		height: 100%;
+	}
+
+	.services__text {
+		display: flex;
+		flex-direction: column;
+		margin: 0 auto;
+	}
+
+	.card__section {
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
+
+	.services--footer__container {
+		display: flex;
+		flex-flow: column wrap;
+		margin: 5px 2px;
+	}
+
+	.footer__content {
+		display: flex;
+		flex-flow: column wrap;
+		margin: 2px 2px;
+	}
+
+	.footer__content ul {
+		margin: 2px 2px;
+	}
+
+	.services-info-img {
+		width: 75%;
+		height: 75%;
+	}
+
+	.footer__content {
+		margin: 1px;
+		padding: 1px;
+	}
+
+
+	.footer__content ul {
+		margin: 2px;
+		display: flex;
+		flex-direction: column;
+		align-self: flex-start;
+		/*margin-left: -5px;*/
+		list-style: square;
+		padding: 2px;
+		margin: 0 auto;
+	}
+
+	.footer__content ul li {
+		padding: 2px;
+		margin: 2px;
+	}
+
+	.services--footer__container {
+		padding: 2px;
+		margin: 2px;
+	}
+}
+
+.nav__container {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 80%;
+}
+.container {
+  display: flex;
+  /*border: 1px solid red;*/
+  width: 80%;
+  justify-content: space-between;
+}
+
+.container li a {
+  display: flex;
+  justify-content: space-around;
+  /*border: 1px solid red;*/
+  text-decoration: none;
+  color: gray;
+}
+
+.logo {
+  align-self: flex-end;
+  justify-self: center;
+}
+
+.hero__container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  width: auto;
+  margin-top: 100px;
+}
+
+.hero__heading {
+  font-size: 80px;
+  text-align: center;
+}
+
+.line {
+  border-top: 1px solid black;
+  margin: 40px;
+}
+
+.btn {
+  border: 2px solid black;
+  width: 200px;
+  text-align: center;
+  box-shadow: 5px 3px 3px black;
+}
+
+.features__container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 20px;
+  text-align: left;
+  margin: 0 70px;
+  /*border: 1px solid red;*/
+}
+
+.features {
+	margin: 0 20px;
+}
+.about {
+	margin: 0 20px;
+}
+.services {
+	margin: 0 20px;
+}
+.product {
+	margin: 0 20px;
+}
+.vision {
+	margin: 0 20px;
+}
+
+.middle__img--container {
+  display: flex;
+  justify-content: center;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items; center;
+  /*border: 1px solid red;*/
+  margin: 10px 20px;
+}
+
+.footer h3 {
+	margin-left: 2%;
+	margin-bottom: 3%;
+}
+
+.footer address {
+	/*border: 1px solid red;*/
+	margin: 5px;
+}
+
+.num {
+	margin: 5px;
+}
+
+.email {
+	margin: 5px;
+}
+
+.copy {
+	margin: 10px;
+	text-align: center;
+}
+
+/*SERVICES PAGE*/
+
+.services-header-img__flexcontainer {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 30px;
+}
+
+.services__container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 20px 20px;
+}
+
```

Comment:
you would wont to set any containers certain sizes so that everything does not continually stretch or shrink on different screen size/resolutions